server/network.go

```diff
@@ -171,22 +174,55 @@ func newLocalNetwork(opts localNetworkOptions) (*localNetwork, error) {
 	}, nil
 }
 
-func createConfigFileString(nodeConfig string, logLevel string, logDir string, dbDir string, pluginDir string, whitelistedSubnets string) (string, error) {
-	var jsonContent map[string]interface{}
-	if err := json.Unmarshal([]byte(nodeConfig), &jsonContent); err != nil {
-		return "", err
+// get the node configs right
+func mergeNodeConfig(defaultConfig string, customConfig string, globalConfig string) (map[string]interface{}, error) {
```

Comment:
I think it would make sense to cache the unmsarshalled representations of `defaultConfig` and `globalConfig`  (that is, store them as `map[string]interface{}`) instead of storing them as `string` and then repeatedly unmarshalling them here.

Reply:
Good suggestion. Fixed.